src/components/RichTooltip/RichTooltip.css

```
@@ -1,10 +1,45 @@
 .RichTooltip {
-  box-shadow: 0 0 4px rgba(0, 0, 0, 0.04), 0 4px 32px rgba(0, 0, 0, 0.16);
-  border-radius: 8px;
-  background: var(--background_content);
+  box-shadow: var(--vkui--elevation3);
+  border-radius: var(--vkui--size_border_radius--regular);
+  background-color: var(
+    --background_content,
+    var(--vkui--color_background_modal)
+  );
   animation: vkui-popper-fadein 0.2s ease;
 }
 
 .RichTooltip__arrow {
-  color: var(--background_content);
+  color: var(--background_content, var(--vkui--color_background_modal));
+}
```

(Тоже глобальный комментарий)
А что, если для стрелочки в `PopperArrow` мы глобально накинем `color: currentColor;` и, соответственно, сократим стили до:

```suggestion
.RichTooltip {
  box-shadow: var(--vkui--elevation3);
  border-radius: var(--vkui--size_border_radius--regular);
  background-color: var(
    --background_content,
    var(--vkui--color_background_modal)
  );
  animation: vkui-popper-fadein 0.2s ease;
  color: var(--background_content, var(--vkui--color_background_modal));
}
```

Или мы так покрасим то, что передается в `content`/деток для тултипов?

---

Угу, там проблема в том, что стрелка должна быть цветом `background`, а `color` для тултипов должен быть разным в зависимости от `mode`

Я создала вот такую [таску](https://github.com/VKCOM/VKUI/issues/2769). Можно будет подсократить код там,  чтобы сильно все в рамках этой задачи не переписывать